### ISSUE TITLE
reload-modified should start with reload-rows

### ISSUE THREAD

**Issue by cool-RR** [saulpw/visidata#2551]
When I use `reload-modified`, I expect to immediately see the updated version of the sheet, and then to have it continually updated as new rows are written. The second part works, but the first part doesn't. I have to wait for a row to be written before I see all the rows that were written between the time I opened the sheet and the time I invoked `reload-modified`. I think there should be a `reload-rows` call at the start of `reload-modified`.

**Comment by cool-RR** [comment]
This is especially important when you use `reload-modified` on a sheet that isn't being written to anymore. You still want to see the latest rows.

**Comment by saulpw** [comment]
You're right, @cool-RR.  I misunderstood this issue at first.  Thanks for reporting it.

**Comment by cool-RR** [comment]
Thank you!
